funnel/views/participant.py

```diff
@@ -106,6 +107,8 @@ def participant_edit(profile, space, participant):
     ((ProposalSpace, ProposalSpaceRedirect), {'name': 'space', 'profile': 'profile'}, 'space'),
     permission='view')
 def participant(profile, space):
+    if space.date_upto and datetime.combine(space.date_upto, datetime.max.time()) < datetime.utcnow():
```

Comment:
1. This uses 23:59:59 of the current day instead of midnight of the next day.
2. This does not account for timezone. Using UTC will make an allowance until 5:30 AM the next day, in India, but will fall short of the day in a Western timezone. Since we do have a `timezone` string column on `space`, use it.